lib/central.js

```diff
@@ -14,6 +14,8 @@ var Cylon = require("cylon");
 
 var UUIDS = [];
 
+var alreadyDiscoveredCharacteristics = {};
```

Comment:
Perhaps might be better as part of the adaptor, aka move down into the constructor, and define as `this.discoveredCharacteristics = {};`